lib/std/time/Date.zig

```diff
@@ -0,0 +1,265 @@
+year: Year = 1970,
+month: Month = .January,
+day_of_month: DayOfMonth = 1, // 1 - 31
+
+const Date = @This();
+const std = @import("../std.zig");
+const s_per_day = std.time.s_per_day;
+
+const seconds_from_epoch_to_y2k = 30 * (365 * s_per_day) + 7 * s_per_day;
+
+pub const Year = i32;
+pub const DayOfMonth = u5; // 1-31
+pub const DayOfYear = u9; // 0-365
+pub const WeekOfYear = u6; // 0-53
+
+pub const Month = enum(u4) {
+    January = 1,
```

Comment:
Enums should be snake_case.

Reply:
The zig style guide has exceptions for "pre-existing naming convention".  Month names are proper nouns and are capitalized.  There are numerous examples all over the standard library of enums following pre-existing conventions.  And since I need the capitalized version for strftime compatibility, it made sense.

https://github.com/ziglang/zig/issues/2101#issuecomment-569857914